R/cdisc_data.R

```diff
@@ -1,6 +1,6 @@
 #' Data input for `teal` app
 #'
-#' @description `r lifecycle::badge("stable")`
+#' `r lifecycle::badge("stable")`\cr
```

Comment:
Might be against the purpose of this PR, but consider this solution to avoid `\cr``_(if it makes sense avoiding `\cr`, it might be a personal thing)_

```suggestion
#' @description `r lifecycle::badge("stable")`
#'
```

Reply:
Okay, we can add `@description` tags back to the `lifecycle badges`. I also noticed that ` @description` is used wherever badges are used in tidyverse packages. Another alternative would be to remove `\cr` so that it appears without a line break.
<img width="344" alt="image" src="https://github.com/insightsengineering/teal.data/assets/6700955/f57b1e01-be7c-4377-be77-a333e7325cb2">

Reply:
Cool finding on tidyverse, we can copy whatever they are doing :grin: 

It seems like they are going with line break

![image](https://github.com/insightsengineering/teal.data/assets/211358/c5d75c58-c888-4286-b956-d9cfd9049ef5)

Reply:
yes, I will add back changes to `teal.slice `as well.

Reply:
[401a3f9](https://github.com/insightsengineering/teal.data/pull/248/commits/401a3f95632ba46fa332f7d6552f18fc1aafa35e)

Reply:
I remember one case where I thought about putting in an empty line instead of a break sign. I decided to leave the break because we a) use them in argument description anyway, b) some functions (then) didn't have `@details`, so I wanted to make sure the `@description` is not broken up, and c) no empty line in docs - no empty line in source.

Reply:
removed other instance of `\cr` [9d07b8c](https://github.com/insightsengineering/teal.data/pull/248/commits/9d07b8ce65a89cbe542a3c6537b612710eec5f23)